packages/lit-dev-content/site/docs/templates/custom-directives.md

```diff
@@ -256,13 +256,13 @@ Async directives often need to subscribe to external resources. To prevent memor
   - When the expression that produced the directive no longer resolves to the same directive.
   After a directive receives a `disconnected` callback, it should release all resources it may have subscribed to during `update` or `render` to prevent memory leaks.
 
-* `reconnected()`: Because DOM subtrees can be temporarily disconnected and then reconnected again later (for example, when DOM is moved or cached for later use) a disconnected directive may need to react to being re-connected. So the `reconnected()` callback should always be implemented alongside `disconnected()`, in order to restore a disconnected directive back to its working state.
+* `reconnected()`: Called when a previously disconnected directive is being returned to use. Because DOM subtrees can be temporarily disconnected and then reconnected again later, a disconnected directive may need to react to being reconnected. Examples of this include when DOM is removed and cached for later use, or when a host element is moved causing a disconnection and reconnection. The `reconnected()` callback should always be implemented alongside `disconnected()`, in order to restore a disconnected directive back to its working state.
 
 * `isConnected`: Reflects the current connection state of the directive.
 
 <div class="alert alert-info">
 
-Note that it is possible for an `AsyncDirective` to continue receiving updates while it is disconnected if its containing tree is re-rendered. As such, `update` and/or `render` should always check the `this.isConnected` flag before subscribing to any long-held resources to prevent memory leaks.
+Note that it is possible for an `AsyncDirective` to continue receiving updates while it is disconnected if its containing tree is re-rendered. Bacause of this, `update` and/or `render` should always check the `this.isConnected` flag before subscribing to any long-held resources to prevent memory leaks.
```

Comment:
Typo in "Bacause", but I think you can lose this too. Just say "`update` and/or `render` should always check the `this.isConnected` flag..." Not sure @arthurevans preference on streamlining the style this way...

Reply:
Seemed abrupt to not connect the two sentences. When I read it without a connecting clause, it's not obvious the 2nd follows from the first.